packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -98,6 +98,74 @@ enum DragAnchor {
 /// widget always displays [child].
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=QzA4c4QHZCY}
+/// 
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// The following example has a [Draggable] widget along with a [DragTarget] 
+/// in a row demonstrating an incremented `acceptedData` integer value when 
+/// you drag the element to the target.
+///
+/// ```dart
+///int acceptedData = 0;
+///
+///Widget build(BuildContext context) {
```

Comment:
There should be at least one space between the comments and the sample code
```suggestion
/// int acceptedData = 0;
///
/// Widget build(BuildContext context) {
```